dashboard/client/src/pages/serve/ServeDeploymentsListPage.tsx

```diff
@@ -89,82 +88,36 @@ export const ServeDeploymentsListPage = () => {
       ) : (
         <React.Fragment>
           <ServeSystemPreview
-            allDeployments={allServeDeployments}
+            allDeployments={serveDeployments}
+            allApplications={serveApplications}
             proxies={proxies}
             serveDetails={serveDetails}
           />
           <CollapsibleSection
-            title="Deployments"
+            title="Applications/Deployments"
```

Comment:
```suggestion
            title="Applications / Deployments"
```

Add spaces to make it less crowded?